.github/workflows/build-macos.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: '1.21.0-rc.4'
+          go-version: '1.21.0'
```

Comment:
Can you change this to `1.21`? My understanding is that `1.21` will pick the latest point release instead of pinning it to 1.21.0.

Reply:
Done,